email_templates/cashEmail/style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```

Comment:
Or really, instead of bothering to have to email it to yourself, I'd just stuff it into codepen and seeing if it looks the way you'd like.